File: Utkarsh_and_Placement_tests.py
import sys
input=sys.stdin.readline

tt = int(input())

for i in range(tt):
    a, b, c = input().split()
    x, y = input().split()

    if a == x:
        print(a)
    elif a == y:
        print(a)
    elif b == x:
        print(b)
    elif b == y:
        print(b)
    elif c == x:
        print(c)
    elif c == y:
        print(c)

File: Utkarsh_and_Placement_tests.cpp
#include <bits/stdc++.h>
using namespace std;
#define ll long long
#define mod 1000000007

void solve()
{
    char pre[3];
    for (int i = 0; i < 3; i++)
        cin >> pre[i];
    char lett[2];
    for (int i = 0; i < 2; i++)
        cin >> lett[i];
    for (int i = 0; i < 3; i++)
    {
        for (int j = 0; j < 2; j++)
        {
            if (pre[i] == lett[j])
            {
                cout << pre[i] << "\n";
                return;
            }
        }
    }
}
int main()
{
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    cout.tie(NULL);
    //#ifndef ONLINE_JUDGE
    // freopen("input.txt", "r", stdin);
    // freopen("output.txt", "w", stdout);
    //#endif

    ll ttt = 1;
    cin >> ttt;
    while (ttt--)
    {
        solve();
    }
    return 0;
}
